LinqToRest/Async/Resources/AbstractionsStrings.cs

```diff
@@ -0,0 +1,58 @@
+﻿// from https://github.com/dotnet/efcore/blob/master/src/EFCore.Abstractions/Properties/AbstractionsStrings.Designer.cs
```

Comment:
Check if this is generated automatically on build and add to .gitignore

Reply:
Decision to not do this, as we would need to add T4 generation to the build, thus adding unessecary complexity, since we don't need to update resources that are copied from EF Core